queries.sql

```diff
@@ -23,3 +23,30 @@ SELECT * FROM animals WHERE name NOT IN ("Gabumon");
 
 -- 8.0 Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <=17.3;
+
+-- Part 2 Questions
+
+-- 1. How many animals are there?
+
+SELECT COUNT(*) FROM animals;
+
```

Comment:
- [ ] It would be better if we `UPDATE` the animals table by setting the species column to digimon for all animals that have a name ending in mon

Reply:
@addod19 , same question as above.

Reply:
Noted. I  have done so.